### PR TITLE
support s02 programme evaluation follow ups

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configurations/coding_plans.py [hunk 1]
```diff
@@ -1222,7 +1258,43 @@ def get_demog_coding_plans(pipeline_name):
                    ],
                    ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value(
                        "kakuma adolescent mothers challenges"),
-                   raw_field_fold_strategy=FoldStrategies.concatenate)
+                   raw_field_fold_strategy=FoldStrategies.concatenate),
+
+    CodingPlan(raw_field="s02_impact_made_raw",
+                                           dataset_name="s02_kakuma_impact_made",
```
Comment (as2388) [inline configurations/coding_plans.py:1264]: indentation

File: configurations/coding_plans.py [hunk 2]
```diff
@@ -1091,6 +1091,42 @@ def get_demog_coding_plans(pipeline_name):
                            ws_code=CodeSchemes.DADAAB_WS_CORRECT_DATASET.get_code_with_match_value(
                                "dadaab adolescent mothers challenges"),
                            raw_field_fold_strategy=FoldStrategies.concatenate),
+
+            CodingPlan(raw_field="s02_impact_made_raw",
+                                       dataset_name="s02_dadaab_impact_made",
```
Comment (as2388) [inline configurations/coding_plans.py]: indentation

**Review by as2388 — APPROVED** [review]

File: configurations/coding_plans.py (hunk 1 above)
Comment (IsaackMwenda) [reply configurations/coding_plans.py:1264]: updated thanks.

File: configurations/coding_plans.py (hunk 2 above)
Comment (IsaackMwenda) [reply configurations/coding_plans.py]: updated thanks.

**Pull request merged**